system/lib/compiler-rt/lib/sanitizer_common/sanitizer_linux.cpp

```diff
@@ -109,6 +109,7 @@ extern struct ps_strings *__ps_strings;
 #include <math.h>
 #include <wasi/api.h>
 #include <wasi/wasi-helpers.h>
+#define SANITIZER_LINUX_USES_64BIT_SYSCALLS 1
```

Comment:
is this a change from upstream?
what are these sanitizer changes for?

Reply:
Because they make direct syscalls, and I removed some of the duplicate "64"-flavored syscalls.   All our syscalls are 64-bit by default .. we don't have `stat` and `stat64` as different syscalls.. which is what `SANITIZER_LINUX_USES_64BIT_SYSCALLS` signals.  

This saves a bit of code here because we don't need `stat64_to_stat` in  this file anymore

Reply:
sounds good. can you put that in the commit description?

Reply:
Actually I reverted that part of the CL and left all the syscall names as is.. much cleaner.